x-pack/test/observability_functional/apps/observability/alerts/index.ts

```diff
@@ -155,6 +161,43 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
           });
         });
       });
+
+      describe('Cell actions', () => {
```

Comment:
This just targets the copy and the filter for value at the moment, the "pop out" functionality is a cell action that comes baked into `EUIDataGrid`, that is to say this tests our *custom* cell actions.